package.json

```diff
@@ -55,6 +56,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-dropzone": "12.0.4",
+    "react-onclickoutside": "^6.12.1",
```

Comment:
just to note for anyone else who looks:  this new library is very widely used. (https://www.npmjs.com/package/react-onclickoutside)